ArkhamDisplay/Data.cs

```diff
@@ -33,9 +33,9 @@ public class DataBlock
     {
         public volatile Theme savedTheme = Theme.Default;
         public volatile Game selectedGame = Game.None;
-        public volatile string[] saveLocations = { "", "", "", "" };
-        public volatile int[] saveIDs = { 0, 0, 0, 0 };
-        public volatile Rect[] windowRects = { new Rect(), new Rect(), new Rect(), new Rect() };
+        public volatile string[] saveLocations = ["", "", "", ""];
```

Comment:
How exactly is this "simplified"? All you've done is replace one kind of bracket with another kind. Both approaches are correct and express the same intent in effectively the exact same way.